src/lightning_app/runners/cloud.py

```diff
@@ -106,6 +106,84 @@ def _to_clean_dict(swagger_object, map_attributes):
 class CloudRuntime(Runtime):
     backend: Union[str, CloudBackend] = "cloud"
 
+    def open(self, name: str, cluster_id: Optional[str] = None):
```

Comment:
No, there are some small differences between the logic of the two methods. The differences will get bigger soon, currently we are only creating an instance because the UI lists instances so we shouldn't need to create an instance at all.

Reply:
Makes sense. It is a better experience.